src/main/java/org/motechproject/mots/domain/JasperTemplate.java

```diff
@@ -68,6 +69,7 @@
   @Fetch(FetchMode.SELECT)
   @Getter
   @Setter
+  @Valid
```

Comment:
you should remove this, because there is no validation in JasperTemplateParameter